docs/src/main/sphinx/sql/show-create-function.md

```diff
@@ -2,20 +2,60 @@
 
 ## Synopsis
 
-```text
+```sql
 SHOW CREATE FUNCTION function_name
 ```
 
 ## Description
 
-Show the SQL statement that creates the specified function.
+Shows the SQL statement that created the specified function.
 
 ## Examples
 
-Show the SQL that can be run to create the `meaning_of_life` function:
+List all SQL routines and plugin functions in the `default` schema of the
+`brain_catalog`:
+
+```sql
+SHOW FUNCTIONS FROM brain_catalog.default
```

Comment:
missing ;

Reply:
Also this should be in show functions docs

Reply:
The show function doc already has examples. Should I replace what is there with these "Answer" examples?

Reply:
Maybe .. but definitely have an answer example in both places